test/plugin/test_filter_record_transformer.rb

```diff
@@ -13,8 +13,10 @@ class RecordTransformerFilterTest < Test::Unit::TestCase
     @hostname = Socket.gethostname.chomp
     @tag = 'test.tag'
     @tag_parts = @tag.split('.')
-    @time = event_time('2010-05-04 03:02:01 UTC')
-    Timecop.freeze(@time)
+
+    @timestamp = Time.parse('2010-05-04 03:02:01 UTC')
+    Timecop.freeze(@timestamp)
+    @event_time = event_time
```

Comment:
Thanks! This should be fixed.

I'm worried about keeping 2 times as it is complicated.
Isn't this sufficient?

```suggestion
    @time = event_time('2010-05-04 03:02:01 UTC')
    Timecop.freeze(@time.to_time)
```